setup.py

```diff
@@ -60,7 +59,7 @@
 
             # local
         ],
-        'pandas': ['pandas'],
+        'pandas': ['pandas<0.20'],
```

Comment:
Why the lock on this version?

Reply:
when we remove that version constraint tests fails with: ```E           AttributeError: 'DataFrame' object has no attribute 'sort'```

Reply:
and fixed it with `_forks.sort_values(by=['creation_time'], inplace=True)`